### ISSUE TITLE
Deprecated import of normalize.css

### ISSUE THREAD

**Issue by JBCSU** [SU-SWS/decanter#184]
**Describe the bug**
`decanter.scss` imports `normalize` from the [normalize.css](https://www.npmjs.com/package/normalize.css) npm package. That's a `.css` file, not `.scss`. webpack's sass loader warns that "Including .css files with @import is non-standard behaviour which will be removed in future versions of LibSass."

**To Reproduce**
1. Configure webpack to use `sass-loader` to process a .scss file which includes `@import "decanter";`. And good luck to you doing that.
2. Use webpack to generate your css from your scss.
3. See this warning:

```DEPRECATION WARNING on line 21, column 8 of ./node_modules/decanter/scss/decanter.scss:
Including .css files with @import is non-standard behaviour which will be removed in future versions of LibSass.
Use a custom importer to maintain this behaviour. Check your implementations documentation on how to create a custom importer.
```

**Expected behavior**
`decanter.scss` should pre-process without warnings.

**Proposed solution**
Find and use an appropriate `normalize.scss`, perhaps [JohnAlbin's](https://www.npmjs.com/package/normalize-scss)?

**Comment by sherakama** [comment]
We will remove the npm dependency on normalize.css and hardcode into our project.

**Comment by JBCSU** [comment]
I tried replacing the `normalize.css` package with [normalize-scss](https://github.com/JohnAlbin/normalize-scss), but that resulted in a conflict on `$base-font-size`: `normalize-scss` defines that variable in px, but we use it in calculation(s) that refer to variables defined in rem, resulting in a unit conflict error when compiling our scss. It was not straightforward to resolve this error. Also, normalize-scss is a version behind normalize.css (7.0.0 vs. 8.0.0). So we decided to remove `normalize.css` from our npm dependencies, rename `normalize.css` as `_normalize.scss`, save it in our scss source directory, and import it from there. We understand and agree that this means we'll have to manually update normalize as new versions become available.
